docker-for-ibm-cloud/administering-swarms.md

```diff
@@ -0,0 +1,266 @@
+---
+description: Administer swarm clusters with Docker EE for IBM Cloud
+keywords: ibm, ibm cloud, logging, iaas, tutorial
+title: Administer swarm clusters with Docker EE for IBM Cloud
+---
+
+Docker Enterprise Edition for IBM Cloud (Beta) comes with a variety of integrations that simplify the swarm administration process. Use the Docker EE for IBM Cloud CLI plug-in (`bx d4ic`) to quickly provision swarm mode clusters and resources. Manage your cluster with the `bx d4ic` plug-in and the Docker EE Universal Control Plane (UCP) browser-accessed interface.
+
+## Create swarms
+Create a Docker EE swarm cluster in IBM Cloud.
+
+> Your beta license allows you to provision up to 20 nodes
+>
+> For beta, your cluster can have a maximum of 20 nodes, up to 14 of which can be worker nodes. If you need more nodes than this, work with your Docker representative to acquire an additional Docker EE license.
+
+Before you begin:
+* [Complete the setup requirements](/docker-for-ibm-cloud/index.md).
+* Make sure that you have the appropriate [IBM Cloud infrastructure permissions](faqs.md).
+* Retrieve your IBM Cloud infrastructure [API username and key](https://knowledgelayer.softlayer.com/procedure/retrieve-your-api-key).
+* Create an SSH key, add the public SSH key to IBM Cloud infrastructure and note its label, and locate the file path of the private SSH key on your machine.
+* Retrieve the Docker EE installation URL that you received in your beta welcome email.
+
+To create a Docker EE for IBM Cloud cluster from the CLI:
+1. Log in to the IBM Cloud CLI. If you have a federated ID, use the `--sso` option.
+   ```bash
```

Comment:
Newlines on both sides of code blocks.

Tip: To line things up nicely along tab stops, add a second space after the period for single-digit numbered lists.  Then your code blocks would be indented 4 spaces instead of 3.